browser/components/ContributionEditor.tsx

```diff
@@ -126,12 +126,8 @@ class ContributionsEditor extends React.Component<Partial<Props>, State> {
       changes.contribution_github_status = field.githubStatusInput.value;
     }
 
-    if (field.githubCommitUrlInput.value.length !== 0) {
-      changes.contribution_commit_url = field.githubCommitUrlInput.value;
-    }
-
     if (field.contributionUrlInput.value.length !== 0) {
-      changes.contribution_commit_url = field.contributionUrlInput.value;
+      changes.contribution_url = field.contributionUrlInput.value;
```

Comment:
whoops. was this a hidden bug?

Reply:
Nope, this is the check to see if anything changed in the form and it now stores into the column that we decided to standardize on.